anvil/src/udev.rs

```diff
@@ -876,14 +871,44 @@ fn render_surface<'a>(
             pointer_element.set_status(cursor_status.clone());
         }
 
-        elements.push(CustomSpaceElements::PointerElement(pointer_element));
+        elements
+            .extend(pointer_element.render_elements(ptr_location.to_physical_precise_round(scale), scale));
```

Comment:
I am not sure if this will work on a multi output setup. The `PointerElement` was a `SpaceElement` before so that it can calculate the `Physical` location on the output. IIRC the pointer location is in `Logical` space.

Reply:
Ok, so I have to substract the output location like before. Will fix that